PWGHF/TableProducer/treeCreatorDplusToPiKPi.cxx

```diff
@@ -79,8 +81,20 @@ DECLARE_SOA_COLUMN(Ct, ct, float);
 // Events
 DECLARE_SOA_COLUMN(IsEventReject, isEventReject, int); //! Event rejection flag
 DECLARE_SOA_COLUMN(RunNumber, runNumber, int);         //! Run number
+DECLARE_SOA_COLUMN(Centrality, centrality, float);     //! Collision centrality (for reco MC)
+// ML scores
+DECLARE_SOA_COLUMN(BkgScore, bkgScore, float); //! Bkg score (for reco MC candidates)
+DECLARE_SOA_COLUMN(FdScore, fdScore, float);   //! FD score (for reco MC candidates)
 } // namespace full
-
+DECLARE_SOA_TABLE(HfCandDpMls, "AOD", "HFCANDDPML",
+                  full::M,
+                  full::Pt,
+                  full::Centrality,
+                  full::BkgScore,
+                  full::FdScore,
+                  hf_cand_3prong::FlagMcMatchRec,
+                  hf_cand_3prong::OriginMcRec,
+                  hf_cand_3prong::FlagMcDecayChanRec)
```

Comment:
Why don't we just put the BDT scores here? In this case we would have a table which is joinable with the `Lite` one and we do not duplicate information